litellm/proxy/_new_secret_config.yaml

```diff
@@ -15,4 +15,12 @@ router_settings:
   redis_password: os.environ/REDIS_PASSWORD
   redis_port: os.environ/REDIS_PORT
```

Comment:
verified this commit passed stable release testing 
<img width="614" alt="Screenshot 2025-03-22 at 7 20 05 PM" src="https://github.com/user-attachments/assets/a1c18c49-c22d-4250-8dec-b3bba0217c83" />